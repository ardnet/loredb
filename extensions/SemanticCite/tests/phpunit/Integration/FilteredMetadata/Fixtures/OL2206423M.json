{"ISBN:0385081308": {"publishers": [{"name": "Anchor Books"}], "pagination": "174 p. :", "identifiers": {"lccn": ["89028542"], "openlibrary": ["OL2206423M"], "isbn_10": ["0385081308"], "librarything": ["16724"], "goodreads": ["1707125"]}, "subtitle": "a collection of Zen & pre-Zen writings", "title": "Zen flesh, Zen bones", "url": "https://openlibrary.org/books/OL2206423M/Zen_flesh_Zen_bones", "classifications": {"dewey_decimal_class": ["294.3/927"], "lc_classifications": ["BQ9264 .R47 1989"]}, "notes": "Reprint. Originally published: Tokyo ; Rutland, Vt. : C.E. Tuttle, 1957.", "number_of_pages": 174, "cover": {"small": "https://covers.openlibrary.org/b/id/4610625-S.jpg", "large": "https://covers.openlibrary.org/b/id/4610625-L.jpg", "medium": "https://covers.openlibrary.org/b/id/4610625-M.jpg"}, "subjects": [{"url": "https://openlibrary.org/subjects/zen_literature", "name": "Zen literature"}, {"url": "https://openlibrary.org/subjects/protected_daisy", "name": "Protected DAISY"}, {"url": "https://openlibrary.org/subjects/accessible_book", "name": "Accessible book"}, {"url": "https://openlibrary.org/subjects/in_library", "name": "In library"}], "publish_date": "1989", "key": "/books/OL2206423M", "authors": [{"url": "https://openlibrary.org/authors/OL418604A/Paul_Reps", "name": "Paul Reps"}], "by_statement": "compiled by Paul Reps.", "publish_places": [{"name": "New York"}], "ebooks": [{"formats": {"djvu": {"url": "https://archive.org/download/zenfleshzenbones00reps/zenfleshzenbones00reps.djvu", "permission": "restricted"}}, "checkedout": true, "borrow_url": "https://openlibrary.org/books/OL2206423M/Zen_flesh_Zen_bones/borrow", "preview_url": "https://archive.org/details/zenfleshzenbones00reps", "availability": "borrow"}]}}