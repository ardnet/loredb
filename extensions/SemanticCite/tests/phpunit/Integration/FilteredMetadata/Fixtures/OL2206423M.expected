{{#scite:OL2206423M
 |title=Zen flesh, Zen bones
 |subtitle=a collection of Zen & pre-Zen writings
 |author=Paul Reps
 |publisher=Anchor Books
 |pubdate=1989
 |pages=174
 |subject=Zen literature;Protected DAISY;Accessible book;In library|+sep=;
 |isbn=0385081308
 |lccn=89028542
 |olid=OL2206423M
 |url=https://openlibrary.org/books/OL2206423M/Zen_flesh_Zen_bones
 |dewey decimal class=294.3/927
 |lc classifications=BQ9264 .R47 1989
 |cover=https://covers.openlibrary.org/b/id/4610625-M.jpg
}}